lib/crypto/atca_crypto_sw.h

```diff
@@ -87,11 +87,20 @@ typedef atca_evp_ctx atcac_pk_ctx;
 #include "wolfssl/wolfcrypt/hmac.h"
 #include "wolfssl/wolfcrypt/sha.h"
 #include "wolfssl/wolfcrypt/sha256.h"
+
 #include "wolfssl/wolfcrypt/error-crypt.h"
 #include "wolfssl/wolfcrypt/asn_public.h"
 #include "wolfssl/wolfcrypt/asn.h"
 #include "wolfssl/wolfcrypt/random.h"
 
+/* these must be defined before including ecc.h because it includes atca_basic.h, which expects atcac_sha2_256_ctx to be setup */
+typedef wc_Sha atcac_sha1_ctx;
+typedef wc_Sha256 atcac_sha2_256_ctx;
+typedef Cmac atcac_aes_cmac_ctx;
+typedef Hmac atcac_hmac_sha256_ctx;
+
+#include "wolfssl/wolfcrypt/ecc.h"
```

Comment:
This is already being included on line 119 after the type definitions. I don't believe changes to this file are necessary anymore.